app/settings.py

```diff
@@ -13,4 +13,7 @@
 
 LOG_LEVEL = os.getenv("LOG_LEVEL", DEFAULT_LOG_LEVEL)
 GOOGLE_APPLICATION_CREDENTIALS = os.getenv("GOOGLE_APPLICATION_CREDENTIALS")
+GCP_PROJECT_NAME = 'bbc-connected-data'
```

Comment:
veeery minor comment, this is referred to as `PROJECT_NAME` in other repos so we could use the same for consistency.

Reply:
Thanks for noticing that!